plugin/src/android/stringsDependency.ts

```diff
@@ -0,0 +1,28 @@
+import { AndroidConfig, ConfigPlugin, withStringsXml } from '@expo/config-plugins'
+import { ResourceXML } from '@expo/config-plugins/build/android/Resources'
```

Comment:
import { ResourceXML } from '@expo/config-plugins/build/android/Resources' the @ is needed in this particular case, because ResourceXML doesn't seems to be available in expo/config-plugins packages. Maybe another way to import it from **expo/config-plugins**?